server/hotrod/src/main/java/org/infinispan/server/hotrod/tx/operation/CommitTransactionOperation.java

```diff
@@ -56,6 +57,7 @@
                return null;
             }
             throwable = throwable.getCause();
+            //TODO handle suspect/outdated topology exceptions
```

Comment:
is there a ticket not to forget the TODO's in this class ?

Reply:
no. I'm using the TODO list in IDEA to keep track of it.